.eslintrc

```diff
@@ -167,6 +167,22 @@
 
         "no-plusplus": "off",
 
-        "no-unused-expressions": "off"
+        "no-unused-expressions": "off",
+		
+		"no-extra-semi" : "off",
+		
+		"wrap-iife" : "off",
+		
+		"no-shadow-restricted-names" : "off",
+		
+		"radix" : "off",
+		
+		"quote-props" : "off",
+		
+		"no-underscore-dangle" : "off",
+		
+		"default-case" : "off",
+		
+		"no-lonely-if" : "off"
```

Comment:
Ezeknél más a behúzás.

Reply:
Nálam ugyan olyan mint minden másik egysoros a fileban

Reply:
A GH csak más behúzásként mutatja, de VS-ben megnyitva már látni fogod, hogy ez a probléma:
![image](https://user-images.githubusercontent.com/1976647/104362399-1120c800-5514-11eb-9441-05b25f0a2840.png)

Reply:
VS-ben nézem és nálam nincs ilyen

![image](https://user-images.githubusercontent.com/36725929/104363100-00248680-5515-11eb-8019-03dec34cfd02.png)

Reply:
De igazad van, csak nálam nem mutatta valamiért, pusholom már

Reply:
Hát ez érdekes volt, nálam a VS-ben a screenshoton is látni, hogy úgy mutatta ahogy kéne lennie, de töröltem előttük, dupla tab és volt változás a commitnál, de vizuálisan nem, nálad ami felajánlotta alap vagy extension?

Reply:
Nem tudom mi ajánlja fel, a Dojo Library-ben ajánlott extensionök vannak nekem csak fent, és továbbra is vegyesen vannak, de ezt már fixelem.